translations/fr.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
These translations should be added via Transifex.

Reply:
For non-en locales.

Reply:
Was added for testing purposes, will revert it.
Thanks!